packages/@coorpacademy-components/src/molecule/table/index.js

```diff
@@ -27,8 +32,8 @@ const createOptionsView = (_options, hasOptions) => {
 };
 
 const Table = (props, context) => {
-  const {rows = [], columns = [], editable = true} = props;
-  const {skin} = context;
+  const {rows = [], columns = [], editable = true, theme = false, headerTitle = ''} = props;
```

Comment:
`theme` prop default value is incorrect: `theme: PropTypes.oneOf([THEMES.COCKPIT]),`